content/aips/RISK-PARAMS-UPDATE-AAVE-V2-ETH.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: Risk Parameter Updates for Aave v2 Ethereum Liquidty Pool 
```

Comment:
please follow proper header formatting (copied from starknet)

> title: Aave StarkNet Phase I - Aave <> StarkNet Bridge deployment/activation by Aave governance
author: Aave on StarkNet
shortDescription: Deployment/activation of the Aave <> StarkNet Bridge
discussions: https://governance.aave.com/t/aave-starknet-phase-i-release/10428
created: 2022-11-18